gms/gossiper.hh

```diff
@@ -548,7 +548,6 @@ public:
     future<> add_local_application_state(std::initializer_list<std::pair<application_state, utils::in<versioned_value>>>);
 
     // Needed by seastar::sharded
-    future<> stop();
```

Comment:
You should also remove `// Needed by seastar::sharded`, which applied top `stop()` only.

Reply:
Removed the comment as well